### PR TITLE
On board simple time tables lni fhi esi bsi agi

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

On_board_simple_time_tables_LNI_FHI_ESI_BSI_AGI

## What is the intent behind these changes?

on board new prisons with correct time tables

### CODE REVIEW

File: src/main/resources/session-template-data/session-data.csv [hunk 1]
```diff
@@ -186,4 +186,29 @@ FNI,Visits CAT A,SOCIAL,3,0,FALSE,14:00:00,16:00:00,2023-04-18,,FRIDAY,FALSE,,FN
 FNI,Visits Main Hall,SOCIAL,30,4,FALSE,14:00:00,16:15:00,2023-04-18,,SATURDAY,FALSE,,FNI_NON_CAT_A_HIGH
 FNI,Visits CAT A,SOCIAL,3,0,FALSE,14:00:00,16:00:00,2023-04-18,,SATURDAY,FALSE,,FNI_CAT_A_HIGH
 FNI,Visits Main Hall,SOCIAL,30,4,FALSE,14:00:00,16:15:00,2023-04-18,,SUNDAY,FALSE,,FNI_NON_CAT_A_HIGH
-FNI,Visits CAT A,SOCIAL,3,0,FALSE,14:00:00,16:00:00,2023-04-18,,SUNDAY,FALSE,,FNI_CAT_A_HIGH
\ No newline at end of file
+FNI,Visits CAT A,SOCIAL,3,0,FALSE,14:00:00,16:00:00,2023-04-18,,SUNDAY,FALSE,,FNI_CAT_A_HIGH
+% Simple Set LNI,FHI,ESI,BSI,AGI
+% LNI
+DHI,Visits Main Hall,SOCIAL,12,3,FALSE,14:15:00,16:15:00,2023-03-21,,TUESDAY,FALSE,,
```
Comment (hutcheonb-moj) [inline src/main/resources/session-template-data/session-data.csv:192]: Put DHI - meant LNI?

File: src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql [hunk 2]
```diff
@@ -26,6 +26,10 @@
         INSERT INTO prison(code, active) SELECT 'MHI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'MHI');
         INSERT INTO prison(code, active) SELECT 'BNI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'BNI');
         INSERT INTO prison(code, active) SELECT 'FNI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'FNI');
+        INSERT INTO prison(code, active) SELECT 'FHI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'FHI');
```
Comment (hutcheonb-moj) [inline src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql:29]: missing prison code LHI

File: src/main/resources/session-template-data/session-data.csv (hunk 1 above)
Comment (AledWEvans) [reply src/main/resources/session-template-data/session-data.csv:192]: looks like I messed up on that one

File: src/main/resources/db/migration/data/R__add_prison_codes.sql [hunk 3]
```diff
@@ -8,3 +8,10 @@ INSERT INTO prison(code, active) SELECT 'EWI', false WHERE NOT EXISTS ( SELECT i
 INSERT INTO prison(code, active) SELECT 'DHI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'DHI');
 INSERT INTO prison(code, active) SELECT 'MHI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'MHI');
 INSERT INTO prison(code, active) SELECT 'BNI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'BNI');
+INSERT INTO prison(code, active) SELECT 'FNI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'FNI');
+INSERT INTO prison(code, active) SELECT 'LNI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'LNI');
+INSERT INTO prison(code, active) SELECT 'FHI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'FHI');
+INSERT INTO prison(code, active) SELECT 'ESI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'ESI');
+INSERT INTO prison(code, active) SELECT 'BSI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'BSI');
+INSERT INTO prison(code, active) SELECT 'AGI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'AGI');
+INSERT INTO prison(code, active) SELECT 'DMI', false WHERE NOT EXISTS ( SELECT id FROM prison WHERE code = 'DMI');
```
Comment (AledWEvans) [inline src/main/resources/db/migration/data/R__add_prison_codes.sql:17]: This has been added for Richard to do historic migration 29+ in past, DMI time table ticket is not ready yet

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

**Pull request merged**